catalogs/catalog-lakehouse-paimon/build.gradle.kts

```diff
@@ -15,56 +15,31 @@ dependencies {
   implementation(project(":common"))
   implementation(project(":core"))
   implementation(libs.bundles.paimon)
+  implementation(libs.bundles.jetty)
+  implementation(libs.bundles.jersey)
   implementation(libs.guava)
   implementation(libs.hadoop2.common) {
     exclude("com.github.spotbugs")
   }
   implementation(libs.hadoop2.hdfs)
-  implementation(libs.hive2.exec) {
-    artifact {
-      classifier = "core"
-    }
-    exclude("com.google.code.findbugs", "jsr305")
-    exclude("com.google.protobuf")
-    exclude("org.apache.avro")
-    exclude("org.apache.calcite")
-    exclude("org.apache.calcite.avatica")
-    exclude("org.apache.curator")
-    exclude("org.apache.hadoop", "hadoop-yarn-server-resourcemanager")
-    exclude("org.apache.logging.log4j")
-    exclude("org.apache.zookeeper")
-    exclude("org.eclipse.jetty.aggregate", "jetty-all")
-    exclude("org.eclipse.jetty.orbit", "javax.servlet")
-    exclude("org.openjdk.jol")
-    exclude("org.pentaho")
-    exclude("org.slf4j")
-  }
-  implementation(libs.hive2.metastore) {
-    exclude("co.cask.tephra")
-    exclude("com.github.spotbugs")
-    exclude("com.google.code.findbugs", "jsr305")
-    exclude("com.tdunning", "json")
-    exclude("javax.transaction", "transaction-api")
-    exclude("org.apache.avro", "avro")
-    exclude("org.apache.hbase")
-    exclude("org.apache.hadoop", "hadoop-yarn-api")
-    exclude("org.apache.hadoop", "hadoop-yarn-server-applicationhistoryservice")
-    exclude("org.apache.hadoop", "hadoop-yarn-server-common")
-    exclude("org.apache.hadoop", "hadoop-yarn-server-resourcemanager")
-    exclude("org.apache.hadoop", "hadoop-yarn-server-web-proxy")
-    exclude("org.apache.logging.log4j")
-    exclude("org.apache.parquet", "parquet-hadoop-bundle")
-    exclude("org.apache.zookeeper")
-    exclude("org.eclipse.jetty.aggregate", "jetty-all")
-    exclude("org.eclipse.jetty.orbit", "javax.servlet")
-    exclude("org.pentaho") // missing dependency
-    exclude("org.slf4j", "slf4j-log4j12")
-    exclude("com.zaxxer", "HikariCP")
-    exclude("com.sun.jersey", "jersey-server")
-  }
+  implementation(libs.hadoop2.mapreduce.client.core)
 
   annotationProcessor(libs.lombok)
   compileOnly(libs.lombok)
+
+  testImplementation(libs.mysql.driver)
```

Comment:
why depend on `postgresql` and `mysql`?

Reply:
`postgresql` is unnecessary.
 
MiniGravitno need to start mysql, without mysql driver, it will fail to start.

Reply:
https://github.com/datastrato/gravitino/blob/main/build.gradle.kts#L169-L171

https://github.com/datastrato/gravitino/blob/main/integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/AbstractIT.java#L139-L147